build.sbt

```diff
@@ -70,10 +64,27 @@ def scalacOptionsFor(scalaBinVersion: String): Seq[String] = scalaBinVersion mat
 }
 
 lazy val mimaSettings = mimaDefaultSettings ++ Seq(
-  mimaBinaryIssueFilters ++= Seq.empty
+  mimaPreviousArtifacts := {
+    val VersionPattern = """^(\d+).(\d+).(\d+)(-.*)?""".r
+    val previousVersions = version.value match {
+      case VersionPattern(epoch, major, minor, _) => (0 until minor.toInt).map(v => s"$epoch.$major.$v")
+      case _ => sys.error(s"Cannot find previous versions for ${version.value}")
+    }
+
+    scalaBinaryVersion.value match {
+      case sbv if sbv.equals(scala213) => Set.empty
+      case _ => previousVersions.toSet.map(previousVersion => organization.value %% name.value % previousVersion)
```

Comment:
This could be a simpler `if` expression 🙂

Reply:
Agree. Let's merge and fix on a separate PR.